src/styles/pages/style.scss

```diff
@@ -0,0 +1,844 @@
+}
+
+.banner__text {
+  font-family: 'Poppins', Arial, sans-serif;
+  font-size: 18px;
+  margin-bottom: 30px;
+  line-height: 1.5;
+}
+
+.banner__stats {
+  display: block;
+  width: 100%;
+  max-width: 800px;
+}
+
+.banner__stat-label {
+  font-family: 'Poppins', Arial, sans-serif;
+  width: 95px;
+  height: 24px;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  letter-spacing: 0px;
+  vertical-align: bottom;
+}
+
+.banner__stat-value {
+  box-shadow: 0px 4px 13px 0px rgba(255, 255, 255, 0.4);
+  font-size: 52px;
+  line-height: 62px;
+  letter-spacing: -0.5px;
+  vertical-align: bottom;
+}
+
+.banner__image {
+  width: 533.499267578125px;
+  height: 641.8035888671875px;
+  top: -0.9px;
+  left: 1030.5px;
+}
+
+.banner__button-group {
+  width: 866.500732421875px;
+  height: 48px;
+  gap: 16px;
+}
+
+.banner__title {
+  background-image: url(src\images\static\our-blog-background.webp);
+}
+
+/* Button Styles */
+.button {
+  display: inline-block;
+  font-family: 'Poppins';
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 1.5;
+  text-decoration: none;
+  padding: 8px 20px;
+  color: white;
+}
+
+.button--primary {
+  background: #0265DC;
+  background-color: #007bff;
+  color: #ffffff;
+}
+
+.button--secondary {
+  background: rgba(208, 215, 251, 0);
+  border: 2px solid rgba(208, 215, 251, 0.17);
+  margin: 0 10px;
+  padding: 10px 20px;
+  background-color: #333;
+  color: #ffffff;
+}
+
+.button--secondary:hover,
+.button--primary:hover {
+  opacity: 0.8;
+}
+
+/* Catalog */
+.catalog {
+  padding: 50px 0;
+  background: #0D0D15;
+  color: #F5F8FB;
+  display: inline-block;
+}
+
+.catalog__header {
+
+  align-items: center;
+  margin-bottom: 30px;
+}
+
+.catalog__title {
+  color: #F8F7FF;
+  font-size: 24px;
+  font-weight: 400;
+  line-height: 32px;
+  letter-spacing: 0px;
+  text-align: left;
+}
+
+.catalog__controls {
+  align-items: center;
+}
+
+.catalog__navigation {
+  align-items: center;
+  margin-left: 20px;
+}
+
+.catalog__navigation button {
+  background: none;
+  border: none;
+  color: #F5F8FB;
+  font-size: 24px;
+  cursor: pointer;
+}
+
+.catalog__navigation button:hover {
+  color: #0265DC;
+}
+
+.catalog__grid {
+  gap: 24px;
+  padding-bottom: 20px;
+  display: inline-block;
+}
+
+.nft-card {
+  width: 300px;
+  background: #1D1C28;
+  padding: 24px;
+  border-radius: 10px;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+  text-align: center;
+  list-style: none;
+}
+
+.nft-card__image {
+  width: 100%;
+  height: auto;
+  border-radius: 10px;
+  margin-bottom: 10px;
+}
+
+.nft-card__badge {
+  padding: 5px 10px;
+  background: #0265DC;
+  color: #F5F8FB;
+  border-radius: 5px;
+  margin-bottom: 10px;
+  top: 10px;
+  left: 10px;
+}
+
+.nft-card__badge--sold-out {
+  background: #EA3829;
+}
+
+.nft-card__price {
+  font-size: 24px;
+  font-weight: bold;
+  color: #F5F8FB;
+  margin-bottom: 10px;
+}
+
+.nft-card__title {
+  font-family: 'Poppins', Arial, sans-serif;
+  font-size: 18px;
```

Comment:
Add correct line-height:
Line height should match the values from the Figma file for proper text alignment and spacing.